app/service/file_svc.py

```diff
@@ -164,6 +177,14 @@ def _get_encryptor(self):
                                    backend=default_backend())
         return Fernet(base64.urlsafe_b64encode(generated_key.derive(bytes(self.get_config('encryption_key'), 'utf-8'))))
 
+    @staticmethod
+    def _load_packers(path):
+        packers = dict()
+        for module in glob.iglob('%s/**.py' % path):
+            packer = import_module(module.replace('/', '.').replace('\\', '.').replace('.py', ''))
```

Comment:
let's look for a more pythonic way to achieve this

Reply:
This seems to be the recommended method, and we use this pattern frequently in the data service and other parts in the code, so I don't believe we have come up with a more clean way of doing it.